packages/core/src/parser/at-value-parser.ts

```diff
@@ -149,7 +150,12 @@ export function parseAtValue(atValue: AtRule): ParseAtValueResult {
       column: start.column + name.length,
       offset: start.offset + name.length,
     };
-    const parsedAtValue = { type: 'valueDeclaration', name, loc: { start, end } } as const;
+    const parsedAtValue = {
+      type: 'valueDeclaration',
+      name,
+      loc: { start, end },
+      definition: atValue.toString(),
```

Comment:
I tried generating JSDoc for value declarations. However, strange hover info appeared in VS Code.

<img width="362" alt="image" src="https://github.com/user-attachments/assets/0a6e1d5e-310f-410f-9ad1-e390e3373dfd" />

I think it would be better to stop generating JSDoc for value declarations rather than displaying strange hover info that confuses users.

Reply:
ref: https://github.com/mizdra/css-modules-kit/pull/179/commits/4cae3a427f619e5c2f172344c5666ced59e45355